src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -1037,9 +1121,11 @@ class ContentSidebar extends PureComponent<Props, State> {
         const { fileId = '' } = this.props;
         if (SidebarUtils.canHaveSidebar(this.props)) {
             if (typeof user === 'undefined') {
-                this.api
```

Comment:
can you update the docstring? There's no more ID, just user. Also that typeof check seems weird

Reply:
I think we can refactor the logic this method outside of this PR. This change was just moving the get request to use an object rather than individual params